lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -385,7 +385,7 @@ def fetch_course_info_with_queries(self, course, sql_queries, mongo_queries):
         self.assertEqual(resp.status_code, 200)
 
     def test_num_queries_instructor_paced(self):
-        self.fetch_course_info_with_queries(self.instructor_paced_course, 26, 4)
+        self.fetch_course_info_with_queries(self.instructor_paced_course, 26, 3)
```

Comment:
@dianakhuang I'm guessing that this is because I'm no longer retrieving the course with depth=2.  I'm not sure why we ever were.  I don't plan on looking into this any more deeply since it is going down and not up.